wrk3.go

```diff
@@ -12,13 +12,26 @@ import (
 	"golang.org/x/time/rate"
 )
 
-type RequestFunc func() error
+type RequestHandler interface {
+	ExecuteRequest(localIndex int) error
+}
+
+type RequestFunc func(int) error
+
+func (reqFunc RequestFunc) ExecuteRequest(localIndex int) error {
+	return reqFunc(localIndex)
+}
+
+var flagsDefined = false
```

Comment:
These shouldn't be global, and I think it was already handled in the command parsing. What happened here?

Reply:
fixed